### PR TITLE
add info about PA-PK-AS-REQ (16)

### PR DESCRIPTION
**Pull request by VLG17**
https://github.com/MicrosoftDocs/windows-itpro-docs/issues/5583

### CODE REVIEW

**Comment by mypil** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: windows/security/threat-protection/auditing/event-4771.md [hunk 1]
```diff
@@ -184,6 +184,7 @@ The most common values:
...
 | 11   | PA-ETYPE-INFO          | The ETYPE-INFO pre-authentication type is sent by the KDC in a KRB-ERROR indicating a requirement for additional pre-authentication. It is usually used to notify a client of which key to use for the encryption of an encrypted timestamp for the purposes of sending a PA-ENC-TIMESTAMP pre-authentication value.<br>Never saw this Pre-Authentication Type in Microsoft Active Directory environment.  |
 | 15   | PA-PK-AS-REP\_OLD      | Used for Smart Card logon authentication.                                                                                                                                                                                                                                                                                                                                                                        |
+| 16   | PA-PK-AS-REQ           | Request sent to KDC in Smart Card authentication scenarios.|
```
Comment (JohanFreelancer9) [inline windows/security/threat-protection/auditing/event-4771.md]:
```suggestion
| 16   | PA-PK-AS-REQ           | The request is sent to KDC in Smart Card authentication scenarios.|
```

**Comment by illfated** [comment]
#sign-off

**Comment by mypil** [comment]
@Dansimp - PR has been copy edited and is ready for final review. This is based on issue #5583

Thank you.

cc: @kenwith

**Pull request merged**